### ISSUE TITLE
OracleQueryRunner.js Bad ( placed

### ISSUE THREAD

**Issue by CarlosGarfio** [typeorm/typeorm#8059]
### Expected Behavior

Connect to postgres.

### Actual Behavior

A compilation error happend at the OracleQueryRunner.js line 670 

```
SyntaxError: Unexpected token ';'
    at wrapSafe (internal/modules/cjs/loader.js:988:16)
    at Module._compile (internal/modules/cjs/loader.js:1036:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/home/code/Projects/click-derecho/salepush/src/driver/oracle/OracleDriver.ts:4:1)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
```

### Steps to Reproduce

1. nest new <project>
2. yarn add @nestjs/typeorm typeorm pg
3. After connecting to the database it should brake

### My Environment

| Dependency          | Version  |
| ---                 | ---      |
| Operating System    |     Ubuntu 20.04.2 LTS      |
| Node.js version     | 14.17.4  | 
| Typescript version  | 4.3.5  | 
| TypeORM version     | ^0.2.36 | 
| @nestjs/TypeORM version     | ^8.0.2 | 

### Additional Context

My solution was simple
I just went from this:
this.replaceCachedTable(oldTable, newTable(;

to this:
this.replaceCachedTable(oldTable, newTable);

then save the file and it work

### Relevant Database Driver(s)

| DB Type              | Reproducible  |
| ---                  | ---           |
| `aurora-data-api`    | no            |
| `aurora-data-api-pg` | no            |
| `better-sqlite3`     | no            |
| `cockroachdb`        | no            |
| `cordova`            | no            |
| `expo`               | no            |
| `mongodb`            | no            |
| `mysql`              | no            |
| `nativescript`       | no            |
| `oracle`             | no            |
| `postgres`           | yes            |
| `react-native`       | no            |
| `sap`                | no            |
| `sqlite`             | no            |
| `sqlite-abstract`    | no            |
| `sqljs`              | no            |
| `sqlserver`          | no            |

### Are you willing to resolve this issue by submitting a Pull Request?

 - ✖️ Yes, I have the time, and I know how to start.
 - ✅ Yes, I have the time, but I don't know how to start. I would need guidance.
 - ✖️ No, I don’t have the time, but I can support (using donations) development.
 - ✖️ No, I don’t have the time and I’m okay to wait for the community / maintainers to resolve this issue.

**Comment by imnotjames** [comment]
I cannot replicate.  Checking the source we don't have that code you're describing.  

I also fetched the package from npm and cannot find the line you're referring to.

Can you provide more information?  What's the URL to the package you're using?  Can you include the relevant typeorm lines from your yarn lockfile

**Comment by imnotjames** [comment]
I tried to replicate this.

The yarn lockfile notes the package artifact as `https://registry.yarnpkg.com/typeorm/-/typeorm-0.2.36.tgz#02295462438b4f1279d26d15d32823c522ae1a04`

Loading the file `OracleQueryRunner.js` (md5 checksum `6420b590ca614fabcb481dee133ce1f3`) there is no occurrence of `(;` and all of the cases of ` this.replaceCachedTable(oldTable, newTable);` look correct.

I also grepped the entire compiled package for `(;` and cannot find any references of that sort.

Cannot replicate.
